WcaOnRails/app/controllers/team_members_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+class TeamMembersController < ApplicationController
+  before_action :authenticate_user!
+  before_action -> { redirect_unless_user(:can_manage_committees?) }, only: [:new, :create]
+  before_action -> { redirect_unless_user(:can_edit_team?, team_from_params) }, only: [:edit, :update]
```

Comment:
Shouldn't it just be `can_edit_team?` for all actions? If someone can manage team why wouldn't he add users? Also I think `can_manage_committees?` has nothing to do here.